tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_jump_opcodes.py

```diff
@@ -0,0 +1,166 @@
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+@pytest.mark.parametrize(
+    "eof_code,expected_hex_bytecode,exception",
+    [
+        pytest.param(
+            # Check that code that uses a new style relative jump succeeds
+            Container(
+                name="EOF1V0008",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH0
+                        + Op.RJUMPI[3]
+                        + Op.RJUMP[3]
+                        + Op.RJUMP[3]
+                        + Op.RJUMP[-6]
+                        + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000E04000400008000015FE10003E00003E00003E0FFFA000bad60A7",
+            None,
+            id="rjump_valid",
+        ),
+        pytest.param(
+            # Check that code that uses a new style conditional jump succeeds
+            Container(
+                name="EOF1V0011",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(1) + Op.RJUMPI[1] + Op.NOOP + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000704000400008000016001E100015B000bad60A7",
+            None,
+            id="rjumpi_valid",
+        ),
+        pytest.param(
+            # Check that jump tables work
+            Container(
+                name="EOF1V0013",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(1)
+                        + Op.RJUMPV[2, 0]
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000D04000400008000016001E2010002000030503050000bad60A7",
+            None,
+            id="jump_tables_valid",
+        ),
+        pytest.param(
+            # Check that jumps into the middle on an opcode are not allowed
+            Container(
+                name="EOF1I0019",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(1)
+                        + Op.RJUMPV[b"\x02\x00\x02\xFF\xFF"]
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000D04000400008000016001E2020002FFFF30503050000bad60A7",
+            EOFException.INVALID_RJUMP_DESTINATION,
+            id="rjump_invalid",
+        ),
+        pytest.param(
```

Comment:
There's no exact test like this in tests/prague/eip7692_eof_v1/eip4200_relative_jumps/test_rjumpi.py, so we can create a new function there with this test, following the format of the rest of the tests in the file.